DTIPrep/DTIPrepRegister.pl

```diff
@@ -1601,12 +1586,12 @@ =head3 register_Preproc($mri_files, $dti_file, $data_dir, ...)
 INPUTS:
   - $mri_files   : hash containing all DTI output information
   - $dti_file    : native DWI file (that will be used as a key
-                    for $mri_files)
-  - $data_dir    : data_dir defined in the config file
-  - $pipelineName: pipeline name (DTIPrepPipeline)
+                    for C<$mri_files>)
+  - $data_dir    : data directory (e.g. C</data/$PROJECT/data>)
+  - $pipelineName: pipeline name (C<DTIPrepPipeline>)
   - $toolName    : tool's name and version
-  - $process_step: processing step ('Preproc' or 'Postproc')
-  - $proc_file   : processed file key ('QCed', 'QCed2'...)
+  - $process_step: processing step (C<'Preproc'> or C<'Postproc'>)
```

Comment:
why have both the C<> and the single quotation?

Reply:
good catch! Done.